core/src/main/java/org/opensearch/sql/utils/DateTimeFormatters.java

```diff
@@ -133,6 +133,30 @@ public class DateTimeFormatters {
           .toFormatter(Locale.ROOT)
           .withResolverStyle(ResolverStyle.STRICT);
 
+  // MMDD
```

Comment:
Perhaps
```suggestion
  // MDD
```

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/232/commits/8388705ffa522ae1eff6581c829dcf4b4ccc5b34